conmon/conmon.c

```diff
@@ -427,28 +483,12 @@ int main(int argc, char *argv[])
 					if (num_read <= 0)
 						goto out;
 
+					buf[num_read] = '\0';
```

Comment:
this is splitting every line into `\0`-separated strings like:
```
[conmon:i]: container PID: 7805
[conmon:i]: read a chunk: (fd=5) '/bin/sh'
[conmon:i]: Processing line: 0, /bin/sh
[conmon:i]: Adding timestamp
[conmon:i]: read a chunk: (fd=5) ': can't create /file: '
[conmon:i]: Processing line: 0, : can't create /file:
[conmon:i]: read a chunk: (fd=5) 'Read-only file system
'
[conmon:i]: Processing line: 0, Read-only file system
[conmon:i]: Processing line: 1,
```
and it's breaking tests (removing this reads the whole line)